src/main/java/io/stargate/sgv2/dynamoapi/dynamo/TableProxy.java

```diff
@@ -22,10 +18,15 @@
 import java.util.Collection;
 import java.util.Date;
 import java.util.List;
-import javax.enterprise.context.ApplicationScoped;
+import java.util.stream.Collectors;
 
-@ApplicationScoped
```

Comment:
This cannot be removed otherwise build won't succeed. I just added it back in the latest commit. Not sure why CI didn't discover it... Maybe there was some cache issue.